earth_enterprise/rpms/build.gradle

```diff
@@ -43,6 +56,21 @@ apply plugin: "nebula.ospackage"
 
 import org.opengee.shell.GeeCommandLine
 
+// Varables used for specifying location of two files
+// 'gee_long_version.txt' and 'gepgcommon'
+def src_dir = "../src/"
+
+// Look for command line paramters that specify how the packages are
```

Comment:
Very minor: Project properties can be passed in `.settings` files, or injected from other projects, in addition to being specified on the command line.  We don't distinguish among these methods.